packages/EasyCore/src/Bridge/Symfony/Traits/InvalidDataGeneratorTrait.php

```diff
@@ -0,0 +1,535 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\Traits;
+
+use Carbon\Carbon;
+use Doctrine\Common\Collections\ArrayCollection;
+
+trait InvalidDataGeneratorTrait
+{
+    /**
+     * @param mixed[] $cases
+     *
+     * @return mixed[]
+     */
+    public function asString(array $cases): array
+    {
+        return \array_map(static function (array $case): array {
+            $case['data'] = \array_map(static function ($value) {
+                return (string)$value;
+            }, $case['data']);
+
+            return $case;
+        }, $cases);
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getArrayCollectionWithFewerItems(string $property, int $minElements): array
```

Comment:
Let's introduce a `getArrayCollectionWithMoreItems` method, just to be consistent with the array case.